src/sparkx/CentralityClasses.py

```diff
@@ -66,12 +66,6 @@ class CentralityClasses:
 
     def __init__(self, events_multiplicity: Union[List[float], np.ndarray],
                  centrality_bins: Union[List[float], np.ndarray]) -> None:
-        if not isinstance(events_multiplicity, (list, np.ndarray)):
```

Comment:
Do the type hints prevent wrong usage of the constructor by themselves?
To my knowledge not. I think here and in other places, you should revert removing instance checks.